src/components/menu/menu-item/menu-item.style.js

```diff
@@ -93,8 +93,13 @@ const StyledMenuItemWrapper = styled.a`
       }
     }
 
+    ${StyledIcon} {
+      width: 16px;
+      height: 16px;
+      margin-right: 5px;
+    }
+
     ${hasSubmenu &&
```

Comment:
Removing the `menuType === "light"` here causes a regression in the `Dark theme` story

![image](https://user-images.githubusercontent.com/44157880/100634789-a03c9c80-3327-11eb-849c-ba7b3002f218.png)